labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -88,7 +88,7 @@ object ListManipulationExercise01 {
    * pattern match or some other method.
    */
   def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
+    iList.zipWithIndex.collect {case (e, i) if ((i % 2) == 0) => e}
```

Comment:
This is super cool!

I interpreted this as get the integers which are odd (note the type is fixed to Int). I noticed that many people interpreted it the way you did. I guess the test cases are ambiguous.